DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fpga_template/src/CMakeLists.txt

```
@@ -0,0 +1,104 @@
+###############################################################################
+### Customize these build variables
+###############################################################################
+set(SOURCE_FILE fpga_template.cpp)
+set(TARGET_NAME fpga_template)
+
+# When targeting a board, the `FPGA_BOARD` will be targeted. Options are:
+#   intel_s10sx_pac:pac_s10
+#   intel_s10sx_pac:pac_s10_usm
+#   intel_a10gx_pac:pac_a10
+set(FPGA_BOARD "intel_s10sx_pac:pac_s10_usm")
```

As (partly) mentioned in the readme:
1/ please add a sentence mentioning that the full path to the BSP may be required
2/ please don't use a usm board as the default board
3/ please allow the users to change the device using cmake .. -DFPGA_DEVICE...
```suggestion
set(FPGA_DEVICE "intel_s10sx_pac:pac_s10_usm")
```